### DISCUSSION TITLE
verbose `tar_renv()`

### DISCUSSION THREAD

**Discussion by ginolhac** [ropensci/targets#328] [Ideas]
Hello,
First of all, thanks for this useful function and also the recent `extras` argument for the `tar_watch()`, nice addition :+1: 
@mattwarkentin I know you were writing this function. Could it be possible to add a `verbose` option to return the actual additional lines that were added to the `_packages.R` ? Or maybe like `renv`, print the changes and maybe ask the user if `y/N` he/she accepts. Because, right now I am using it blindly, look at the `diff` and pay attention to the packages declaration, re-run etc...

Thanks again!

**Comment by wlandau** [comment]
I can understand how `renv` might motivate this, but I think `_packages.R` is less sensitive than a lockfile, and I would prefer not to add a menu or print to the R console.

**Reply by ginolhac** [reply]
fair enough, the functionality is here anyway
